package.json

```
@@ -13,6 +13,7 @@
     "debug": "npm run build -- --watch --debug",
     "licenses": "license-report --config=./.license-report-config.json > dist/licenses.html",
     "lint": "eslint",
+    "addons-linter": "addons-linter dist",
```

If the dist does not exist or it is not a Firefox build, the command will not work as expected. What do you think about the change, and also rename the command to follow `lint`:
```suggestion
    "lint:build": "npm run build firefox && addons-linter dist",
```